xm-commons-lep/src/main/java/com/icthh/xm/commons/lep/api/BaseLepContext.java

```diff
@@ -0,0 +1,34 @@
+package com.icthh.xm.commons.lep.api;
+
+import com.icthh.xm.commons.lep.BaseProceedingLep;
+import com.icthh.xm.commons.lep.spring.LepThreadHelper;
+import com.icthh.xm.commons.lep.spring.lepservice.LepServiceFactory;
+import com.icthh.xm.commons.security.XmAuthenticationContext;
+import com.icthh.xm.commons.tenant.TenantContext;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class BaseLepContext {
+
+    public Object commons;
+    public Object inArgs;
+    public BaseProceedingLep lep;
+    public LepThreadHelper thread;
+    public XmAuthenticationContext authContext;
+    public TenantContext tenantContext;
+    @Deprecated(forRemoval = true)
+    public Object methodResult;
```

Comment:
Maybe add some information which class or field should be used instead of this one?

Reply:
just don't use, not alternatives